rdr_service/cron_default.yaml

```diff
@@ -157,7 +157,7 @@ cron:
 - description: Genomic CVL W3SR Workflow
```

Comment:
Can you throw something like (scheduled weekly, runs bi-weekly) in the description?

Reply:
Sure